assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.js

```diff
@@ -53,25 +52,28 @@ export default function WidgetAreaRenderer( { slug, totalAreas } ) {
 	// A combined CTA will span the combined width of all widgets that it was
 	// combined from.
 	const {
-		gridClassNames,
+		gridColumnWidths,
 		overrideComponents,
 	} = combineWidgets( widgets, widgetStates, {
-		classNames,
 		columnWidths,
 		rowIndexes,
 	} );
 
 	// Render all widgets.
 	const widgetsOutput = widgets.map( ( widget, i ) => (
-		<WidgetRenderer
-			gridClassName={ classnames( gridClassNames[ i ] ) }
-			OverrideComponent={ overrideComponents[ i ] ? () => {
-				const { Component, metadata } = overrideComponents[ i ];
-				return <Component { ...metadata } />;
-			} : undefined }
-			key={ widget.slug }
-			slug={ widget.slug }
-		/>
+		<WidgetWrapper
```

Comment:
Maybe let's use a more specific name here? How about `WidgetCellWrapper`? While technically it can also wrap the widget in a div that hides it, the primary purpose is to wrap it in a grid cell.